app/connector/telegram/src/main/resources/META-INF/syndesis/connector/telegram.json

```diff
@@ -72,15 +72,15 @@
       "type": "MAVEN"
     }
   ],
-  "description": "Send and receive messages to/from a Telegram chat",
+  "description": "Receive and send messageses.",
```

Comment:
I think this is wrong, otherwise this looks good.

Reply:
Glad you caught that!